eng/Packages.Data.props

```diff
@@ -64,14 +64,16 @@
     <PackageReference Update="System.Numerics.Vectors" Version="4.5.0" />
     <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="4.6.0" />
     <PackageReference Update="System.Reflection.TypeExtensions" Version="4.5.1" />
-    <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.2" />
+    <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />
```

Comment:
Any particular reason for the version update?

Reply:
I think this was required by System.Linq.Async